packages/device_info/device_info/lib/device_info.dart

```diff
@@ -33,292 +30,5 @@ class DeviceInfoPlugin {
   ///
   /// See: https://developer.apple.com/documentation/uikit/uidevice
   Future<IosDeviceInfo> get iosInfo async =>
-      _cachedIosDeviceInfo ??= IosDeviceInfo._fromMap(
-          await channel.invokeMapMethod<String, dynamic>('getIosDeviceInfo'));
-}
-
-/// Information derived from `android.os.Build`.
-///
-/// See: https://developer.android.com/reference/android/os/Build.html
-class AndroidDeviceInfo {
```

Comment:
All these classes that you moved from device_info to device_info_platform_interface need to be re-exported (so they're still part of the public API). 

Look at the current documentation of device_info to know which classes you need to ensure are visible from the plugin: https://pub.dev/documentation/device_info/latest/device_info/device_info-library.html

Reply:
Completely forgot about that. :( . I've done the changes now.